src/m365/spfx/commands/project/project-upgrade/rules/FN006004_CFG_PS_developer.ts

```diff
@@ -3,8 +3,10 @@ import { Project } from '../../project-model/index.js';
 import { Finding } from '../../report-model/index.js';
 
 export class FN006004_CFG_PS_developer extends JsonRule {
-  constructor(private version?: string) {
+  private version?: string;
+  constructor(options?: { version?: string }) {
```

Comment:
Add an empty line between the property and constructor for readability. Apply consistently in all affected files.